catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/forms/filter/TransformVisitorXmlTest.java

```diff
@@ -76,6 +77,18 @@ public void testVisitPropertyIsEqualTo() throws Exception {
         .withData(DEPTH_PROP, DEPTH_VAL);
   }
 
+  @Test
+  public void testVisitDWithin() throws Exception {
+    getRootJsonFilterNode("spatial-distance-ops", "DWithin.json").accept(visitor);
+    forElement(visitor.getResult())
+        .withBinding(FilterType.class)
+        .forElement(FilterType::getSpatialOps)
+        .withBinding(DistanceBufferType.class)
+        .verifyExpressionOrAny(DistanceBufferType::getExpressionOrAny)
+        .withData("anyGeo", "WKT()");
+    // TODO verify DistanceBufferType::getDistance
```

Comment:
❗️ Address TODO comment

Reply:
@vinamartin No, I updated the unit tests